recipes/quilt/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "quilt" %}
+{% set version = "3.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/quiltdata/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ded69d039949de8293b0c98e95cd0a958eb7e68c93106b1f35254dd4691685c4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - appdirs>=1.4.0
+    - aws-requests-auth >=0.4.2
+    - boto3 >=1.8.0 
+    - elasticsearch==6.3.1
+    - jsonlines==1.2.0
+    - numpy>=1.14.0                   # required by pandas, but missing from its dependencies.
+    - packaging>=16.8
+    - pandas>=0.19.2
+    - pyarrow ==0.13              # as of 7/5/19: linux/circleci bugs on 0.14
+    - requests>=2.12.4
+    - ruamel.yaml<=0.15.70
+    - tqdm>=4.26.0
+    - urllib3==1.24          # required by requests
+    - xattr>=0.9.6
+    - humanize
+
+test:
+  imports:
```

Comment:
i'm checking to see how to run [our standard CircleCI tests here](https://github.com/quiltdata/quilt/blob/master/.circleci/config.yml#L33)